roles/glance/templates/etc/serverspec/glance_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+[['glance-api.conf', 640], ['glance-api-paste.ini', 640], ['glance-registry.conf', 640], ['glance-registry-paste.ini', 640], ['glance-swift-store.conf', 640], ['policy.json', 644]].each do |file| 
+  describe file("/etc/glance/#{file[0]}") do
+    it { should be_mode file[1] }
+    it { should be_owned_by 'glance' }
+    it { should be_grouped_into 'glance' }
+  end
+end
+
+describe file('/var/log/glance/glance-api.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'glance' }
+  it { should be_grouped_into 'adm' }
+end
+
+describe file('/var/log/glance/glance-manage.log') do
+  it { should be_mode 644 }
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'adm' }
+end
+
+describe file('/var/log/glance/glance-registry.log') do
+  it { should be_mode 644}
+  it { should be_owned_by 'glance' }
+  it { should be_grouped_into 'adm'}
+end
+
+describe file('/etc/logrotate.d/glance') do
+  file_contents = [ '# Generated by Ansible.',
+                    '# Local modifications will be overwritten.',
+                    '',
+                    '/var/log/glance/*.log',
+                    '{',
+                    '  daily',
+                    '  missingok',
+                    '  rotate',
+                    '  compress',
+                    '}']
+  file_contents.each do |file_line|
+    it { should contain file_line}
+  end
+end
+
+describe command("cat /etc/glance/glance-registry.conf|grep debug") do
+  its(:stdout) { should eq "debug = True\n" }
```

Comment:
First, why use cat just to grep? You can grep a file directly without catting first.

Second, why not use the method used by other server specs, e.g. https://github.com/blueboxgroup/ursula/pull/1593/files#diff-30e9feac5304c5dac9cf568e5747c52eR43